api/src/Entity/CampCollaboration.php

```diff
@@ -86,7 +86,7 @@
 #[ORM\Index(columns: ['status'])]
 class CampCollaboration extends BaseEntity implements BelongsToCampInterface {
     public const ITEM_NORMALIZATION_CONTEXT = [
-        'groups' => ['read', 'CampCollaboration:Camp', 'CampCollaboration:User'],
+        'groups' => ['read', 'CampCollaboration:Camp', 'CampCollaboration:User', 'User:Profile'],
```

Comment:
Revert this and create a filter by camp or collaboration or user on the profiles endpoint